.tav.yml

```diff
@@ -186,44 +186,53 @@ graphql:
     commands: node test/instrumentation/modules/graphql.test.js
 
 express:
-  versions:
-    mode: latest-minors
-    include: '>=4 <5'
-  commands:
-    - node test/instrumentation/modules/express/basic.test.js
-    - node test/instrumentation/modules/express/capture-exceptions-off.test.js
-    - node test/instrumentation/modules/express/capture-exceptions-on.test.js
-    - node test/instrumentation/modules/express/set-framework.test.js
+  - versions:
+      mode: latest-minors
+      include: '>=5 <6'
+    node: '>=18'
+    commands:
+      - node test/instrumentation/modules/express/basic.test.js
+      - node test/instrumentation/modules/express/capture-exceptions-off.test.js
+      - node test/instrumentation/modules/express/capture-exceptions-on.test.js
+      - node test/instrumentation/modules/express/set-framework.test.js
+  - versions:
+      mode: latest-minors
+      include: '>=4 <5'
+    commands:
+      - node test/instrumentation/modules/express/basic.test.js
+      - node test/instrumentation/modules/express/capture-exceptions-off.test.js
+      - node test/instrumentation/modules/express/capture-exceptions-on.test.js
+      - node test/instrumentation/modules/express/set-framework.test.js
 
+# XXX: check if there is a new version that supports express@5 and add it as
+# a new entry to the test matrix. Also use the new versions mode to select `latest-minors`
```

Comment:
```suggestion
```

I think we can drop this XXX. https://www.npmjs.com/package/apollo-server-express says the package is now deprecated. The last release was about a year ago.

Eventually, I'd be fine with us dropping testing of apollo-server-express. It (and other graphql-related libs) provide enough maintenance hardship.